packages/ng/tooltip/trigger/tooltip-trigger.directive.ts

```diff
@@ -256,6 +256,10 @@ export class LuTooltipTriggerDirective implements AfterContentInit, OnDestroy {
 		if (!isNativelyFocusableTag && !hasATabIndex) {
 			this.#renderer.setAttribute(this.#host.nativeElement, 'tabindex', tabindex.toString());
 		}
+
+		if (!isNativelyFocusableTag && !this.luTooltipWhenEllipsis() && !this.luTooltipOnlyForDisplay) {
+			this.#renderer.setAttribute(this.#host.nativeElement, 'role', 'button');
```

Comment:
If we add the button role to an element, should it be typed to avoid triggering forms?

Reply:
I confirm, the role does not add behavior.